docs/reference/vppcontainers/twocontainers.rst

```diff
@@ -0,0 +1,117 @@
+.. _twocontainers:
+
+.. toctree::
+
+
+=====================================
+VPP with two Containers using Vagrant
+=====================================
+
+
+Overview
+________
+
+This section will cover how to install a VM (Virtual Machine) for Vagrant, and how to use containers inside that VM.
```

Comment:
This section will describe how to install a Virtual Machine (VM) for Vagrant, and install containers inside that VM.